### PR TITLE
--profile argument were consolidated to AWS ?= ... definition

### PR DESCRIPTION
**Pull request by gloovers**
## Description
There were consolidated `--profile=$(AWS_PROFILE)` and `--region=$(AWS_REGION)` arguments from each of `aws cli` invokation to one `AWS $= ...` macros.

It allows framework users:
- to have more readable code;
- to exclude human issues

## Testing Done
image

image

image

### CODE REVIEW

File: aws/main.mk [hunk 1]
```diff
@@ -9,15 +9,21 @@ endif
...
 # $(AWS_ARGS) definition see in .infra/icmk/aws/localstack.mk
-AWS ?= $(DOCKER) run -v $(HOME)/.aws/:/root/.aws -i amazon/aws-cli:2.0.40 $(AWS_ARGS)
-CMD_AWS_LOGS_TAIL = @$(AWS) logs tail --profile $(AWS_PROFILE) $(SERVICE_NAME) --follow
-CMD_AWS_EC2_IMPORT_KEY_PAIR = @$(AWS) ec2 import-key-pair  --key-name="$(EC2_KEY_PAIR_NAME)" --profile $(AWS_PROFILE) --public-key-material="$(SSH_PUBLIC_KEY_BASE64)"
+AWS ?= $(DOCKER) run \
+	-v $(HOME)/.aws/:/root/.aws \
+	-i \
+	-e AWS_PROFILE="$(AWS_PROFILE)" \
+	-e AWS_REGION="$(AWS_REGION)" \
+	amazon/aws-cli:2.0.40 $(AWS_ARGS)
```
Comment (AutomationD) [inline aws/main.mk:23]: @gloovers I'd say we have a good change to lock down the version via variable (similar to `TERRAFORM_VERSION`) (not related to features in this PR)

**Review by AutomationD — CHANGES_REQUESTED** [review]
Looks great! Thank you, @gloovers!
Let's add a `Testing Done` section into the PR that outlines basic tests (that you've been doing). It could be a command-log sequence or asciicinema.

File: aws/main.mk (hunk 1 above)
Comment (gloovers) [reply aws/main.mk:23]: I agree, though about the same, it will be done in another PR

**Review by AutomationD — APPROVED** [review]
Looks good, thank you!

**Pull request merged**